scss/layout/stanford-department.scss

```diff
@@ -0,0 +1,139 @@
+@charset "UTF-8";
+
+// My Config.
+@import "config/config";
+//// Decanter config.
+@import "decanter/core/utilities/decanter-utilities";
+// Grab our mixins.
+@import "utilities/utilities";
+
+// Pop out the tabs from the dom flow so they hover over.
+@include hero-banner-node;
+
+// Hide the lockup with CSS as we do need it to re-appear in
+// a different place.
+
+#header {
+  .block--lockup {
+    @include grid-media($media-md) {
+      visibility: hidden;
```

Comment:
This needs be be hidden but since all other elements in this area are absolutely positioned I need it to take up space. I'm not 100% but I think this is ok for screen readers.